R/read_mow_data.R

```diff
@@ -53,12 +53,14 @@ read_mow_data <- function(filename, n_max = Inf) {
                                              "latitude", crs_lambert, crs_wgs84)
 
     # Define mapping for mow supported values
+    wt <- "T\u00B0"
+    Encoding(wt) <- "UTF8"
     variable_mapping <- list("DO" = "dissolved_oxygen",
                              "Turb_YSI" = "turbidity_ySI",
                              "Turb_SG5" = "turbidity_SG5",
                              "Turb_SG25" = "turbidity_SG25",
                              "Cond" = "conductivity",
-                             "T\u00B0" = "water_temperature")
+                             wt = "water_temperature")
```

Comment:
Unfortunately in this way you create a slot called `"wt"`instead of using the value of `wt`. Typical example of use of NSE in R.
Example:
```r
> damiano <- "hans"
> list(damiano = 1)
$damiano
[1] 1
```

I am pushing now a solution.

Reply:
Good catch! Thanks for spotting and fixing this 👍